.circleci/config.yml

```diff
@@ -230,14 +230,23 @@ workflows:
           filters:
             tags:
               only: /^v.*/
+      - architect/push-to-docker:
+          image: "quay.io/giantswarm/aws-operator"
+          username_envar: "QUAY_USERNAME"
+          password_envar: "QUAY_PASSWORD"
```

Comment:
At some point it would be cool to rename those. But at least it's clear where they are used.